project-koku/koku
# COST-747 & COST-750: Create mat views & instance type api for GCP.

What I have tackled in this pr:
1) Initially the cost api was completed before the nise work. Therefore, we used a tmp csv file to populate the data for the api unit tests. However, now that the nise work is finished we have replaced that temporary csv file work with the built in nise feature. 
2) I have created the materialize views for the gcp instance type api
3) I have created the gcp instance type api. 

## How to test

Run `make load-test-customer-data` then hit the gcp instance-type endpoints to see if they come up.
